### DISCUSSION TITLE
Reproducible and maintainable data science code with Kedro at PyCon US 2021

### DISCUSSION THREAD

**Discussion by yetudada** [kedro-org/kedro#791] [Idea]
[![PyCon US 21](https://user-images.githubusercontent.com/43755008/122439925-fc999d80-cf93-11eb-9340-18e0229d194a.png)](https://youtu.be/JLTYNPoK7nw)

I recently presented a talk on [_Reproducible and maintainable data science code with Kedro_]( https://youtu.be/JLTYNPoK7nw). Here's the blurb and check it out: 

> Code produced by data scientists is under attack! There are a growing series of conference talks, Medium blog posts and business stakeholders telling a story of how changing business objectives are driving interest in production-level code. Production-level code is considered time-consuming to produce and limiting for the experimentation process needed to create amazing models. You're going to follow a workflow that deconstructs your experimentation workflow in a Jupyter notebook and helps you create production-ready ML pipelines. The talk is focused on an open source Python framework, called `kedro` that emphasises creating reproducible, maintainable and modular data science code.

**Comment by lorenabalan** [comment]
I've literally just watched this today, it's so good! The progression with Jupyter notebooks was 🔥 🔥  !!
